src/main/java/be/syntra/devshop/DevshopBack/controllers/CartController.java

```diff
@@ -3,13 +3,10 @@
 import be.syntra.devshop.DevshopBack.models.CartDto;
 import be.syntra.devshop.DevshopBack.services.CartService;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
-@RequestMapping("/carts")
+@RequestMapping("/users/{userId}/carts")
```

Comment:
avoid double /'s in class controller. @Requestmapping("/users) for a CartController seems weird, but okay. Move the "/{userId}/carts to the corresponding Postmapping

Reply:
fixed